### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_HighContrast : Keyboard focus is not clearly visible on checkboxes under "nullable" header after applying solarized dark theme.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6167]
[Check out Keros! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS2.4.7; #MAS1.4.3; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Test Pass Name: A11y_AzureDataStudio
MAS: MAS1.4.3
Version: 1.9.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS :Windows_NT x64 10.0.18362

**Prequisites:**
Apply High Contrast Black/White mode.

**Additional Details:**
Tools Name: ColorContrast, Keyboard
MAS References: MAS1.4.3,MAS2.4.7

**Steps to Reproduce:**
1. Install the app Azure Data Studio.
2. Go to setting and select color theme Solarized dark theme
3. Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.
4. Select any tree view item from left panel using arrow key and press enter.
5. Observed that keyboard focus is not clearly visible on checkboxes under "Nullable" header after applying solarized dark theme.

**Actual:**
While navigating checkboxes under "Nullable" header ,keyboard focus is not clearly visible on these checkboxes after applying Solarized dark theme.

**Expected:**
On applying solarized dark theme, keyboard focus should be clearly visible on checkboxes under "Nullable" header .

**User Impact:**
Low- vision user may face difficulty while traversing through the application as the focus for the few of the controls is not visible.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference :
MAS1.4.3:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
**MAS2.4.7:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

Attachment for Reference:
[6167_[Accessibility]A11y_AzureDataStudio_MapDataSrcObjects_ColorContrast .pptx](https://github.com/microsoft/azuredatastudio/files/3328342/6167_.Accessibility.A11y_AzureDataStudio_MapDataSrcObjects_ColorContrast.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
Non-Default theme issue.

**Comment by Charles-Gagnon** [comment]
Closing this as it is for a non-default theme
